doc/upgrade-cluster.md

```diff
@@ -221,6 +221,32 @@ INFO[5930] error syncing cluster version, upgrade failed. failed to sync bookkee
 ...
 ```
 
+### Handling Upgrade from one version to another
+
+When upgrading from version X to version Y (where both X and Y < 0.7), we need to ensure that we do not modify any field other than the version field while triggering the upgrade. Altering of any other parameter can be handled either before triggering the upgrade or after the upgrade has successfully completed. Same holds true even when the both X and Y >= 0.7.
+
+However, when a user wants to upgrade Pravega from a version < 0.7 to a version >= 0.7, there are configuration changes that must be made to Pravega manifest. This is because from version 0.7 onwards, we no longer use Rocks DB for the Pravega Segmenstore. These changes need to be made either with the upgrade request or prior to starting the upgrade. These configuration changes are needed to make sure we have setup memory and other pravega parameters appropriately for segment store streaming cache to work as expected after moving to version 0.7.
```

Comment:
Remove all this text. User documentation should be concise, easy to read and to the point.
Please create new section after "Trigger an Upgrade" with the title "Upgrade Guide"
Under this section you should have :
a) Upgrades till Pravega 0.6

b) Upgrade to Pravega 0.7
Here mention the additional configuration changes needed when triggering the upgrade.
See this for reference:
https://bookkeeper.apache.org/docs/latest/admin/upgrade/#upgrade-guides